src/core/build_target.go

```diff
@@ -1773,6 +1773,20 @@ func (target *BuildTarget) GetFileContent(state *BuildState) (string, error) {
 	return ReplaceSequences(state, target, target.FileContent)
 }
 
+// HasLinks returns true if the outputs are meant to be linked somewhere (i.e. via symlinks).
+// This check is useful in deciding whether this target should be downloaded during remote execution or not.
+func (target *BuildTarget) HasLinks(state *BuildState) bool {
```

Comment:
Not really sure this is the best place for it, but I reckon it makes sense since certain labels have a meaning and functionality attached to them